tests/test_core/test_typesbase.py

```diff
@@ -61,7 +63,7 @@ def test_simple_struct_array_set_and_get(self):
                 int minute;
                 int second;
             } [20]
-        """)
+        """, _ffi)
```

Comment:
Should use dist.load() to retrieve an instance of ffi like we do in normal code.